landoapi/models/landing.py

```diff
@@ -40,39 +49,34 @@ def create(cls, revision_id, phabricator_api_key=None, save=True):
 
         repo = phab.get_revision_repo(revision)
 
+        # save landing to make sure we've got the callback
+        landing = cls(
+            revision_id=revision_id,
+        ).save()
+
         trans = TransplantClient()
+        callback = '%s/landings/%s/update' % (
+            os.getenv('HOST_URL'), landing.id
```

Comment:
Discussed on IRC, I'm fine with both keeping the variable as-is and with changing it.